fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/ami/FetchAmiJob.java

```diff
@@ -123,6 +134,8 @@ private void runOn(String account, String region) {
                                     .withEventId(EVENT_ID)
                                     .withType(OUTDATED_TAUPAGE)
                                     .withInstanceId(instance.getInstanceId())
+                                    .withAccountId(taupageYaml.map(data -> (String) data.get(APPLICATION_ID)).map(StringUtils::trimToNull).orElse(null))
```

Comment:
Jesus...that happens if you rely on autocompletions first result to be correct 😞